.github/workflows/tf-validation.yml

```
@@ -0,0 +1,43 @@
+# github-actions workflow
+# this test will just run terraform without salt
+name: e2e tests
+
+on: [pull_request]
+
+jobs:
+  terraform-deploy-netweaver-hana:
+    runs-on: self-hosted 
+
+    steps:
+    - uses: actions/checkout@v2
+
+       
+    - name: terraform apply netweaver and hana
+      env: #
+        QEMU_URI: ${{ secrets.qemu_uri }}
+        SOURCE_IMAGE: ${{ secrets.source_image }}
+     
+      run: |
+        cp .github/hana-netweaver-tf-only.tfvars libvirt/terraform.tfvars
+        cd libvirt
+        export TF_VAR_qemu_uri="$QEMU_URI"
+        export TF_VAR_source_image="$SOURCE_IMAGE"
```

You could avoid defining the vars and then exporting them again with another name: you can just name the env vars in the top section with the TF_VAR prefix.
```suggestion
      env: #
        TF_VAR_qemu_uri: ${{ secrets.qemu_uri }}
        TF_VAR_source_image: ${{ secrets.source_image }}
     
      run: |
        cp .github/hana-netweaver-tf-only.tfvars libvirt/terraform.tfvars
        cd libvirt
```

---

done